upbound/xgql
# Add Google Cloud Trace Exporter option

### Description of your changes



Resolves #76 

I have:


### How has this code been tested



1. Run xgql with the following flags on a GKE cluster where [application default credentials available](https://cloud.google.com/docs/authentication/production#automatically) and xgql has permissions to write traces to cloud trace. 
        - --trace-backend=gcp
        - --trace-ratio=0.1
2. Check the GCP cloud trace console and verify xgql traces are available there.
